src/LVMGenerator.h

```diff
@@ -391,6 +391,12 @@ class LVMGenerator : protected RamVisitor<void, size_t> {
     }
 
     void visitAggregate(const RamAggregate& aggregate, size_t exitAddress) override {
+        // TODO xiaowen: The aggregate operation is now written in a less efficient way
```

Comment:
```suggestion
        // TODO (xiaowen): The aggregate operation is now written in a less efficient way
```
Even better would be to make an issue for this.